src/database/DataTypes/Sensor.js

```diff
@@ -5,9 +5,199 @@
 
 import Realm from 'realm';
 
+import { NativeModules } from 'react-native';
+import { generateUUID } from 'react-native-database';
+
+const MANUFACTURER_ID = 307;
+
+const getCommand = (manufacturerID, command, param) => {
+  const commands = {
+    307: {
+      BLINK: '*blink',
+      RESET_ADV_FREQ: '*sadv1000',
+      RESET_LOG_FREQ: `*lint${param}`,
+      DL_LOGS: '*logall',
+    },
+  };
+  return commands[manufacturerID][command];
+};
+
 export class Sensor extends Realm.Object {
-  // TODO
-  // Does this need to be a class ??
+  /**
+   * Scans for all devices matching the passed manufacturer ID,
+   * returning their advertisement data. Updates or creates
+   * any found sensors.
+   */
+  static async scanForSensors(database) {
+    try {
+      const { BleManager } = NativeModules;
+      const result = await BleManager.getDevices(MANUFACTURER_ID, '');
```

Comment:
perhaps it's better to have an overloaded method in native (with, omitting mac address)

Reply:
5de77427f7b82a154ab6eb0a0cfdf812acb57ffd

Reply:
7add042deeb8f01ae9fb27d714297eb888c6746d turns out you cant overload native module methods - so have reverted this. Want your opinion: Create two methods with different names in java and create a javascript method (probably in `Sensor` which would have an `if` to determine which method should be called?